lib/library.go

```diff
@@ -383,6 +383,24 @@ func SaveAccountAndLogin(accountData, password, configJSON, subaccountData *C.ch
 	return makeJSONResponse(nil)
 }
 
+// DeleteAccount delete account's database and delete account from multiaccount database
+//export DeleteAccount
+func DeleteAccount(addressChars *C.char) *C.char {
```

Comment:
Can we use RPC in this case? I believe the consensus is to use RPC when poissble

Reply:
hm, yes I think we can